github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.31.yaml

```diff
@@ -0,0 +1,66 @@
+---
+presubmits:
+  k8snetworkplumbingwg/kubemacpool:
+    - name: pull-kubemacpool-unit-test-v0.31
+      branches:
+        - release-v0.31
+      annotations:
+        fork-per-release: "true"
```

Comment:
This only makes sense for the main branch

Reply:
Done